packages/Rakefile

```diff
@@ -50,8 +67,8 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
     url = "https://github.com/groonga/groonga/releases/download/v#{@version}/"
     if target_namespace == :apt and !target.end_with?("-i386")
       url << "#{target}-amd64.tar.gz"
-    else
-      url << "#{target}.tar.gz"
```

Comment:
Does this still work with `:yum` `target_namespace`?

Reply:
Ah...
Yes. This still works with yum and apt for x64.

I added this in e2847366adb9e7819e5a55cc66374dba662e3f3b